assemblies/installing-odh-v1.adoc

```diff
@@ -0,0 +1,30 @@
+:_module-type: ASSEMBLY
+//pv2hash: 
+
+ifdef::context[:parent-context: {context}]
+
+:context: installv1
+
+[id="installing-odh-v1_{context}"]
+= Installing Open Data Hub (v1)
```

Comment:
Suggest "Installing Open Data Hub version 1" or "Installing version 1 of Open Data Hub" rather than the parentheses.